### ISSUE TITLE
Error : Duplication du nom de colonnes "TYPE_WATER" #921

### ISSUE THREAD

**Issue by yoannlmr** [orbisgis/geoclimate#942]
Hi, I tested on Window for Papeete, French Polynesia... And I have the exactly same issue... #921 
I know you fixed this in the current master, but as a noob I don't found the solution in the topic... or in the current master... Could you help me ?

The goal will be to repeat this for Bora-Bora, Tahiti and Moorea cities ! :)

Thank's you !

```json
{
"description": "Processing OSM data",
"input": {
"locations": [
"Papeete, Îles du Vent, Polynésie française, 98714"
]
},
"output": {
"folder": "C:/Users/yolam/Documents/Geoclimate/Results"
},
"parameters": {
"rsu_indicators": {
"indicatorUse": [
"LCZ",
"TEB",
"UTRF"
],
"svfSimplified": true,
"estimateHeight": true
},
"grid_indicators": {
"x_size": 100,
"y_size": 100,
"rowCol": false,
"output" : "geojson",
"indicators" :["BUILDING_FRACTION", "BUILDING_HEIGHT", "BUILDING_POP",
"BUILDING_TYPE_FRACTION", "WATER_FRACTION", "VEGETATION_FRACTION",
"ROAD_FRACTION", "IMPERVIOUS_FRACTION", "FREE_EXTERNAL_FACADE_DENSITY",
"BUILDING_HEIGHT_WEIGHTED", "BUILDING_SURFACE_DENSITY",
"SEA_LAND_FRACTION", "ASPECT_RATIO", "SVF",
"HEIGHT_OF_ROUGHNESS_ELEMENTS", "TERRAIN_ROUGHNESS_CLASS",
"UTRF_AREA_FRACTION", "UTRF_FLOOR_AREA_FRACTION",
"LCZ_PRIMARY"]
}
}
}

**Comment by j3r3m1** [comment]
Hi @yoannlmr , can you tell which version of GeoClimate do you use ?

**Comment by yoannlmr** [comment]
Hi @j3r3m1, I use GeoClimate 1.0.1.

Available here :
https://nightly.link/orbisgis/geoclimate/workflows/CI_snapshot/master/geoclimate-1.0.1-SNAPSHOT

**Comment by j3r3m1** [comment]
Under investigation.

What are the GeoClimate outputs you are interested in ? If you do not need the grid indicators ones, using the configuration file below should work.

```json
{
"description": "Processing OSM data",
"input": {
"locations": [
"Papeete, Îles du Vent, Polynésie française, 98714"
]
},
"output": {
"folder": "C:/Users/yolam/Documents/Geoclimate/Results"
},
"parameters": {
"rsu_indicators": {
"indicatorUse": [
"LCZ",
"TEB",
"UTRF"
],
"svfSimplified": true,
"estimateHeight": true
}
}
}

**Comment by j3r3m1** [comment]
I cannot reproduce the error you are describing, it runs well without error for me.

If you have downloaded the last GeoClimate snapshot as described in your previous comment, are you sure you are using the right jar file when executing GeoClimate ?

**Comment by yoannlmr** [comment]
I just tried your technique whitout grid indicators, it's work, so it's perfect !

Like I said... I'm sure of nothing ahah ! I tried with other cities in France and it's worked... But when I tried with Papeete, it was not. I don't undertand what's happened... 

Thank's for you help, Outputs with your script will be enough for my work. I hope that I will not have any problem for other cities of French Polynesia !

Thank you very much for your patience and responsiveness. And good job for GeoClimate, it's a very impressive job !
